src/main/java/eu/europa/ted/efx/xpath/XPathScriptGenerator.java

```diff
@@ -333,6 +333,12 @@ public BooleanExpression composeComparisonOperation(Expression leftOperand, Stri
         leftOperand.script + " " + operators.get(operator) + " " + rightOperand.script);
   }
 
+  @Override
+  public BooleanExpression composeSequenceEqualFunction(ListExpressionBase one,
+      ListExpressionBase two) {
+    return new BooleanExpression("not((" + one.script + " except " + two.script + ", " + two.script + " except " + one.script + "))");
```

Comment:
This XPath will not work for literal values: "except" only takes 2 **node** sequences (https://www.w3.org/TR/xpath20/#combining_seq).
"(1,2,3) except (2,3,4)" causes the error: "Required item type of second operand of 'except' is node()"

Furthermore, "except" is based on node identity (https://www.w3.org/TR/xpath-datamodel/#node-identity)
For the XML:
```
<a>
  <b>
    <c>1</c>
  </b>
  <b>
    <c>2</c>
  </b>
  <b>
    <c>1</c>
  </b>
</a>
```

The XPath `/a/b/c except /a/b[1]/c` will return 2 nodes, corresponding to the second and third "c".

I think the following XPath should work as expected:
`deep-equal(sort(<one>), sort(<two>))`